static/js/contribute.js

```diff
@@ -0,0 +1,42 @@
+//Make the DIV element draggagle:
+dragElement(document.getElementById(("contribute-box")));
+
+function dragElement(elmnt) {
+  var pos1 = 0, pos2 = 0, pos3 = 0, pos4 = 0;
+  if (document.getElementById(elmnt.id + "-tabs")) {
+    /* if present, the header is where you move the DIV from: */
+    document.getElementById(elmnt.id + "-tabs").onmousedown = dragMouseDown;
+  } else {
+    /* otherwise, move the DIV from anywhere inside the DIV: */
+    elmnt.onmousedown = dragMouseDown;
+  }
+
+  function dragMouseDown(e) {
+    e = e || window.event;
```

Comment:
When do we need `window.event`? That's a non-standard Internet Explorer API which normally wouldn't be needed on any supported browser.